resources/templates/Default/ViewHelper.rst

```diff
@@ -3,12 +3,15 @@
 .. _{headlineIdentifier}:
 
 {headlineDecoration}
-{headline}
+<f:format.raw>{headline}</f:format.raw>
```

Comment:
Question: Why format.raw?

Reply:
because we are dealing with rst here and not with html, however fluid is always escapting signs that need to be escaped in html